### ISSUE TITLE
UTRs task2 Thodburg TSS

### ISSUE THREAD

**Issue by ValWood** [pombase/pombase-chado#820]
For this dataset we need the EMM (condition)  files for vegetative growth:

These are
https://www.pombase.org/data/external_datasets/Thodberg_GSE110976/
GSE110976_EMM_exp_minus.bw | 2018-05-25 06:52 | 4.0M |  
GSE110976_EMM_exp_plus.bw | 2018-05-25 06:52 | 4.0M |  

These are bigwig files.
@kimrutherford  what sort of conversion do I need to see these in Artemis?

**Comment by kimrutherford** [comment]
> what sort of conversion do I need to see these in Artemis?

From the [manual](https://sanger-pathogens.github.io/Artemis/Artemis/artemis-manual.html#_Toc532219059) it sounds like Artemis supports reading WIG format these days.  I'll make .wig files and I'll give it a go.  I'll let you know how it goes.  We might need to change the chomosome IDs in the wig file to match what's in the contig files.

**Comment by ValWood** [comment]
> We might need to change the chomosome IDs in the wig file to match what's in the contig files.

or split into 3 separate files? that is what I have bee doing with the gff, because I wasn't sure that in Artemis there was any knowledge of 'chromosome number' (i'm probably wrong, would be useful to know).

**Comment by kimrutherford** [comment]
I managed to load a track into Artemis but all I got was a flat line and slow scrolling.  I'll try to make a GFF file instead.

**Comment by kimrutherford** [comment]
> I'll try to make a GFF file instead.

I've made three GFF files for each chromosome.  There are lots of small features from the bigwig files so I've used a score cut off to trim things down.  I've used a cut off of 0.5, 5 and 50.  The corresponding files end in "-0.5.gff", "-5.gff" and "-50.gff".  I don't know which cut is best.  Let me know if these are useful.  Is it worth creating files for the smaller chromosome pieces (MT etc.)?

https://curation.pombase.org/kmr44/thodberg_for_artemis.tar.gz
